stubs/click-default-group/METADATA.toml

```diff
@@ -0,0 +1,5 @@
+version = "1.2.*"
+requires = ["click"]
```

Comment:
Unfortunately "non-types dependencies" are not yet possible in typeshed -- see #5768. We've recently been working towards _making_ them possible, but this work is not yet complete. (It will hopefully be complete soon, but I don't have a timeline for you, I'm afraid.)

The best workaround we currently have is to do something like what `types-pycocotools` currently does for `numpy`:

https://github.com/python/typeshed/blob/5c3f10e1df3d6ad86d9d725aa42d0512d7b86ad9/stubs/pycocotools/pycocotools/cocoeval.pyi#L1-L10

_But_ that seems like quite a poor solution in this situation, given how fundamental `click` is to these stubs. I'm not really sure what to suggest :( an alternative would be to mark this as "deferred" for now, until we've fixed #5768.